### DISCUSSION TITLE
关于补祯滤镜的问题

### DISCUSSION THREAD

**Discussion by sqq135** [hooke007/mpv_PlayKit#362] [libmpv]
我用的是rife_std.vpy滤镜，放在profile自动加载，是vf = vapoursynth="~~/vs/rife_std.vpy"，虽然tab的界面也会出现这个滤镜，但是控制台总是显示这个
![屏幕截图 2023-09-06 133339](https://github.com/hooke007/MPV_lazy/assets/78015121/db20f45f-487b-49e9-b284-f0f1a46f8717)
，换成其他滤镜也有各种unsupport，
取消掉这个滤镜在input里面设置快捷键vf set vapoursynth="~~/vs/rife_std.vpy" ，也同样出现图片中的问题，是否是缺少某个vapoursynth.00啊，求指教

**Comment by hooke007** [comment]
仅凭此无法判断
请展示完整的输出内容。
你用的版本也过时了。

**Reply by sqq135** [reply]
不好意思，没有说清楚。用的是一个基于[mpv.net_CM]的分支，是这个已经过时了吗，感觉右键还挺好用的，如果想要切换成lazy版，是否需要重新开始整页面、快捷键的啊？

**Reply by hooke007** [reply]
如发布帖所述，该项目不再维护。

**Reply by sqq135** [reply]
那想迁移过去有没有人开发类似的界面啊，无意冒犯，只是习惯了界面

**Reply by hooke007** [reply]
哈姆雷特。在我眼里差不多
你可以自行体验。

**Reply by sqq135** [reply]
先把熟悉的快捷键迁了过来，关于lazy版有几个不明白的基础问题。
1、发布页的包，vsMega、noBin是不是不用管，安装方式是不是安装标准版后覆盖安装vsLite和hook版，然后编辑input和mpv.conf就可以了？
2、无论是按照上面的方法安装和只安装标准版（只是打开和解压，没设为默认播放器），按shift+3（#或者说SHARP）时，都会出现
![屏幕截图 2023-09-07 012312](https://github.com/hooke007/MPV_lazy/assets/78015121/66ec83e3-8864-4b6a-9a70-553024ab38fd)
shift+1或者2都是好的，是不是安装vs时错了，实在小白，discussion也没找到相关，不好意思。
3、更新时是否可以只迁移portable_config，还是说里面的conf文件

**Reply by sqq135** [reply]
这个显示台怎么复制内容不太会，抱歉只能发图片

**Reply by hooke007** [reply]
显卡线程被限制为1了，根据之前的案例，是垃圾amd的驱动问题

**Reply by hooke007** [reply]
1-标准包提供的功能已在文档中记录。你提到的功能不需要其它包
建议完整阅读说明先

3-个人设置迁移参考 https://github.com/hooke007/MPV_lazy/discussions/201

**Reply by sqq135** [reply]
应该如何解决呢？顺便问下设置自动加载滤镜
这样的步骤
在profile.conf对应的下面加上vf set vapoursynth="~~/vs/MEMC_MVT_LQ.vpy" 
然后滤镜放在vs文件夹中
不能自动加载，但是设置快捷键vf set vapoursynth="~~/vs/MEMC_MVT_STD.vpy"却可以

**Reply by sqq135** [reply]
还有github这样显示，是不是有问题啊
![image](https://github.com/hooke007/MPV_lazy/assets/78015121/786f7bb3-b8e0-44a5-9b08-499b4cbe0b30)

**Reply by hooke007** [reply]
input和mpvconf语法不同，
mpv里填的是基本是选项
input里的填写的主要是命令，参见对应文档 https://hooke007.github.io/unofficial/mpv_input.html

**Reply by hooke007** [reply]
github的显示是markdown语法，你复制的东西里存在符号，会触发md语法渲染，你可以自行了解一下md的书写方式

**Reply by sqq135** [reply]
感谢感谢，已经解决。
最后一个剩下的问题，驱动的问题该怎么解决啊
> 显卡线程被限制为1了，根据之前的案例，是垃圾amd的驱动问题

**Reply by hooke007** [reply]
换驱动 https://github.com/HomeOfVapourSynthEvolution/VapourSynth-RIFE-ncnn-Vulkan/issues/40
